brayns/common/scene/Model.cpp

```diff
@@ -105,10 +105,59 @@ void Model::addModel(ModelPtr model, const Transformations& transform)
     _modelsDirty = true;
 }
 
+uint64_t Model::addSDFGeometry(const size_t materialId, const SDFGeometry& geom,
+                               const std::vector<size_t>& neighbourIndices)
+{
+    _sdfGeometryData.isDirty = true;
+    const uint32_t geomIdx = _sdfGeometryData.geometries.size();
+    _sdfGeometryData.geometryIndices[materialId].push_back(geomIdx);
+
+    assert(_sdfGeometryData.geometries.size() ==
+           _sdfGeometryData.neighbours.size());
+
+    _sdfGeometryData.neighbours.push_back(neighbourIndices);
+    _sdfGeometryData.geometries.push_back(geom);
+
+    switch (geom.type)
+    {
+    case brayns::SDFType::Sphere:
+    {
+        _bounds.merge(geom.center - Vector3f(geom.radius));
+        _bounds.merge(geom.center + Vector3f(geom.radius));
+        break;
+    }
+    case brayns::SDFType::Pill:
```

Comment:
use default: instead?

Reply:
I added a throw on the default case so that you do not forget to set the bounds if you add a new geometry type.